border-center-for-journalists/border-hub-website
# Add instagram icon

#### What does this PR do?
* Remove part of the author component rendering 
###### View extra context for more information
* Add instagram to authors query
* Draws Instagram data to authors component if exist
* Add instagram icon and font from icomoon
#### Where should the reviewer start from?
`components/notice/authors.js`
`containers/normalNote.js`
`containers/specialnote.js`
`pages/index.js `
#### Any background context you want to provide?
I deleted part of the component because the data retrieved by prismic it was always the same type of object 
```
{…}
-> email: Object { text: null }
​-> facebook: Object { text: null }
​-> name: Object { text: "María Idalia Gómez" }
​-> twitter: Object { text: null }
-> instagram: Object { text: null }
​-> user_picture: Object { url: null }
```
Just changed the alignment of the Author name and picture_image by the provided `align` prop to keep the design on special notices and normal notices

#### How should this be tested?
run `gatsby develop` 
Go to `localhost:8000/debaten-funcionarios-periodista-y-activista-la-relacion` in your browser
it should show the instagram icon and user
#### TIP: Delete the fake instagram from that notice before merge

#### Questions?
###### Yes I checked all the notices
